packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.html

```diff
@@ -46,6 +46,6 @@
         </template>
       </div>
     </section>
-    <div class={backdropClass}></div>
+    <div class={backdropClass} onfocus={lastElementOnFocus} tabindex="0"></div>
```

Comment:
Can we call this a really obvious name? like `resetFocusToTop` ?

That way I don't even have to go look at what the method does to understand why this is here.

Reply:
function name lastElementOnFocus change to resetFocusToTop